spec/models/podcast_spec.rb

```diff
@@ -10,8 +10,14 @@
   it { is_expected.to validate_presence_of(:feed_url) }
 
   context "when callbacks are triggered after save" do
+    let(:podcast) { build(:podcast) }
+
+    before do
+      allow(Podcasts::BustCacheWorker).to receive(:perform_async)
+    end
     it "triggers cache busting on save" do
-      expect { build(:podcast).save }.to have_enqueued_job.on_queue("podcasts_bust_cache").once
+      podcast.save
+      expect(Podcasts::BustCacheWorker).to have_received(:perform_async).with(podcast.path)
```

Comment:
just a note: this should probably check the job is in the queue with the sidekiq test helpers, instead of using mocks

Reply:
Got it, I will change it ;)

Reply:
Lets do this spec like [you did the one above](https://github.com/thepracticaldev/dev.to/pull/5421/files#diff-e78bfcfd62a254776b9c62050d3e10a1R55) and I think this will be ready to go!!!

Reply:
done!